tests/boardfarm_plugins/boardfarm_sample_plugin/devices/prplmesh_docker.py

```diff
@@ -57,11 +57,16 @@ def __init__(self, *args, **kwargs):
                         + " --verbose --detach --force --name {}".format(self.name) \
                         + " --mac {}".format(self.mac)
 
-        if self.role == "controller":
```

Comment:
Controller alone doesn't need to be supported IMO, we don't support running a standalone controller without a local agent in prplmesh anyway.